packages/orbit-components/src/TextLink/__tests__/index.test.js

```diff
@@ -1,73 +1,68 @@
 // @flow
 import * as React from "react";
-import { shallow } from "enzyme";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 
 import TextLink from "../index";
 import ChevronRight from "../../icons/ChevronRight";
 
-describe("TextLink", () => {
-  const title = "My text link";
-  const href = "https://kiwi.com";
-  const onClick = jest.fn();
-  const type = "primary";
-  const tabIndex = "-1";
-  const dataTest = "test";
-  const rel = "nofollow";
+const title = "My text link";
+const dataTest = "test";
 
-  const component = shallow(
-    <TextLink
-      onClick={onClick}
-      href={href}
-      type={type}
-      rel={rel}
-      external
-      icon={<ChevronRight />}
-      tabIndex={tabIndex}
-      dataTest={dataTest}
-    >
-      {title}
-    </TextLink>,
-  );
-
-  const componentWithoutHref = shallow(<TextLink>{title}</TextLink>);
-
-  it("should contain a children", () => {
-    expect(component.children().exists()).toBe(true);
-  });
-  it("should contain an href", () => {
-    expect(component.render().prop("href")).toBe(href);
-  });
-  it("should have data-test", () => {
-    expect(component.render().prop("data-test")).toBe(dataTest);
-  });
-  it("should have noopener in attribute", () => {
-    expect(component.render().prop("rel").split(" ").includes("noopener")).toBe(true);
-  });
-  it("should have noreferrer in attribute", () => {
-    expect(component.render().prop("rel").split(" ").includes("noreferrer")).toBe(true);
-  });
-  it("should have rel values in the rel attribute", () => {
-    expect(component.render().prop("rel").split(" ").includes(rel)).toBe(true);
-  });
-  it("should have tabindex", () => {
-    expect(component.render().prop("tabindex")).toBe(tabIndex);
-  });
-  it("should contain an external href", () => {
-    expect(component.render().prop("target")).toBe("_blank");
-  });
-  it("should contain SVG", () => {
-    expect(component.find("ChevronRight").exists()).toBe(true);
+describe("#TextLink", () => {
+  it("should be focusable and have button role", () => {
+    render(<TextLink>{title}</TextLink>);
```

Comment:
Considering that you're testing the role, it's best to set it explicitly, otherwise changing the default `asComponent` will needlessly break the test.

```suggestion
    render(<TextLink asComponent="button">{title}</TextLink>);
```

Reply:
Make total sense! 🙂